tests/rptest/services/redpanda.py

```diff
@@ -2379,26 +2380,45 @@ def raise_on_crash(self):
         error message, rather than having failures on "timeouts" which
         are actually redpanda crashes.
         """
+
+        allow_list = []
+        if log_allow_list:
+            for a in log_allow_list:
+                if not isinstance(a, re.Pattern):
+                    a = re.compile(a)
+                allow_list.append(a)
+
+        def is_allowed_log_line(line: str) -> bool:
+            for a in allow_list:
+                if a.search(line) is not None:
+                    return True
+            return False
+
         crashes = []
         for node in self.nodes:
             self.logger.info(
                 f"Scanning node {node.account.hostname} log for errors...")
 
             crash_log = None
             for line in node.account.ssh_capture(
-                    f"grep -e SEGV -e Segmentation\ fault -e [Aa]ssert -e Sanitizer {RedpandaService.STDOUT_STDERR_CAPTURE} || true",
+                    f"grep -e SEGV -e Segmentation\\ fault -e [Aa]ssert -e Sanitizer {RedpandaService.STDOUT_STDERR_CAPTURE} || true",
```

Comment:
Curious about this extra `\`?

Reply:
Oh, right. Yeh, you can do `fr"grep...` for raw-f-string (i think). anyway, lgtm.